models/question/Question.js

```diff
@@ -371,4 +371,66 @@ QuestionSchema.statics.getQuestionJSONByAges = function (id, is_percent, callbac
     });
 };
 
+
+QuestionSchema.statics.createFilterGraph = function (data, callback) {
+  // Collects data from user and format the data
+  // Creates graphs with the ratio of the each data
+
+  if (!data.firstQuestionId || !data.secondQuestionId) return callback('bad_request', null)
+  const yQuestionId = data.firstQuestionId;
```

Comment:
Bir de callback bad_request te null a gerek yok, zaten olkayan field ı null dönüyor. Ben bazen tek argüman dönerken null yazıyorum, güzel göründüğü için gerek yok yoksa